tests/Unit.Tests/SharedViewModels/FinanceViewModel.Tests.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
+using GovUk.Education.SearchAndCompare.Domain.Models;
+using GovUk.Education.SearchAndCompare.Domain.Models.Joins;
+using GovUk.Education.SearchAndCompare.UI.ViewModels;
+using NUnit.Framework;
+
+namespace SearchAndCompareUI.Tests.Unit.Tests.SharedViewModels
+{
+    [Category("SharedFinanceViewModel")]
+    public class FinanceViewModelTests
+    {
+        private readonly FeeCaps _feeCaps = new FeeCaps {
+            Id = 0,
+            StartYear = 1990,
+            EndYear = 2010,
+            UkFees = 9001,
+            EuFees = 9002,
+            InternationalFees = 800000
+        };
+
+        private readonly Course _emptyCourse = new Course ();
+
+        private readonly SubjectFunding _subjectFunding = new SubjectFunding
+        {
+            Scholarship = 2,
+            BursaryFirst = 1
+        };
+
+        [Test]
+        [TestCase("Physics", "Biology")]
+        [TestCase("Mathematics", "Biology")]
+        [TestCase("Music", "Drama")]
+        public void EarlyCareerPaymentsFlag_On_Course_With_Maths_Or_Physics_And_Null_Setting_Should_Be_False(string subject1, string subject2)
+        {
+            var subjects = new List<string> { subject1, subject2 };
+            _subjectFunding.EarlyCareerPayments = null;
+            _emptyCourse.CourseSubjects = SetupSUbjects(subjects, _subjectFunding);
+            var sut = new GovUk.Education.SearchAndCompare.UI.Shared.ViewModels.FinanceViewModel(_emptyCourse, _feeCaps);
+            sut.HasEarlyCareerPayments.Should().Be(false);
+        }
+        [Test]
```

Comment:
minor - lack of newline between tests makes the file a bit harder to read

Reply:
Done